src/scandipwa/app/plugin/event-dispatches/Purchase.plugin.js

```diff
@@ -0,0 +1,59 @@
+import Event, { EVENT_GTM_PURCHASE } from '../../util/Event';
+
+export const PLACE_ORDER_MUTATION = 's_placeOrder';
+
+class PurchasePlugin {
```

Comment:
I'd swap places of the two methods of this class, because the lower one is called first and the upper one is called second. But it is not severe. Otherwise - everything is just as requested. Nice job!